gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```

Comment:
Doesn't that set it to 64 megabytes? On CI we [set it to 2048m](https://github.com/mozilla/glean/blob/5066c88a60560607413690e7329721d25fdd656c/.circleci/config.yml#L166).

At the very least this should have a comment what it does and why so low

Reply:
That's automatically generated by the Gradle thing, a-c has the same. I'm not sure we should change this,

Reply:
Ah well, then, let's leave it untouched.